SampleApp/src/main.cpp

```diff
@@ -91,14 +91,14 @@ int main(int argc, char* argv[]) {
             }
         }
     } else {
-#if defined(KWD_SENSORY) || defined(KWD_GPIO) || defined(KWD_HID)
+#if defined(KWD_SENSORY)
         if (argc < 3) {
             ConsolePrinter::simplePrint(
                 "USAGE: " + std::string(argv[0]) +
 #ifndef XMOS_AVS_TESTS
-                " <path_to_AlexaClientSDKConfig.json> <path_to_inputs_folder> [log_level]");
-#else
                 " <path_to_AlexaClientSDKConfig.json> <path_to_inputs_folder> [log_level] [op_point]");
+#else
```

Comment:
These changes are only meant to skip building the Sensory KWD engine if SENSORY_KEY_WORD_DETECTOR_FLAG is not set. This flag is never set so the KWD engine is never built. I could remove completely the steps rather than protect them with a flag. This is actually the approach used in the latest version of SDK in the alexa repo:
https://github.com/alexa/avs-device-sdk/blob/master/tools/Install/setup.sh

I don't mind to remove completely the steps to build and install the Sensory KWD engine, it will make the scripts more readable too. I would choose this solution only we are sure we won't use Sensory in the future.

Reply:
Sorry, I wasn't specific enough in my original comment.  The changes on lines 99 and 100 of the old version and lines 100 and 101 of the new version change the information contained in the usage message.  The usage message in the new version with `XMOS_AVS_TESTS` not defined now matches the usage message in the old version with `XMOS_AVS_TESTS` defined.  Is that the intention?

Reply:
Sorry for the late reply. Those lines are correcting a bug in the original code. The changes are of no use now as Sensory is not working and therefore we can't use the XMOS AVS tests, but I thought about including them anyway.

The new line 99 is used when XMOS_AVS_TESTS is not set, whereas the new line 101 is used when that flag is set. To be honest the distinction is not really necessary as when we compile with KWD_SENSORY, XMOS_AVS_TESTS is always set as well. I added the ifdef for XMOS_AVS_TESTS just to make it easier to remove the XMOS_AVS_TESTS feature in the future.